File: Android/EmotionsApp/EmotionsApp/Models/DTOs/Auth/SignUpRequest.swift
//
//  SignUpRequest.swift
//  EmotionsApp
//
//  Created by Муса Зарифянов on 08.11.2025.
//

import Foundation

struct SignUpRequest: Codable, Sendable {
    private var username: String
    private var email: String
    private var password: String
    
    init(username: String, email: String, password: String) {
        self.username = username
        self.email = email
        self.password = password
    }
}

File: Android/app/src/main/java/com/example/emotionapp/domain/entities/SignUpRequest.kt
package com.example.emotionapp.domain.entities

data class SignUpRequest(
    val username: String,
    val email: String,
    val password: String,
    /** Возраст обязателен на бэкенде; в UI пока нет поля — отправляем безопасное значение по умолчанию. */
    val age: Int = 25
)
